core/CronArchive.php

```diff
@@ -991,7 +990,7 @@ public function isThereAValidArchiveForPeriod($idSite, $period, $date, $segment
         if ($isTodayIncluded
             && !$isLast
         ) {
-            return [false, null];
+            return [false, $date];
```

Comment:
we need to return the date here, as otherwise the date used for archiving will be null here: 
https://github.com/matomo-org/matomo/blob/9969201da8fb875d6953e87b6e818b743c35e791/core/CronArchive.php#L902

Reply:
@sgiehl did you notice this by fixing some test or was something not working? I assume there was some core:archive used with specifically today?

Reply:
see the linked issue https://github.com/matomo-org/matomo/issues/15727